### DISCUSSION TITLE
Drag & drop multiple items of M2M field at once

### DISCUSSION THREAD

**Discussion by YuryYCasumo** [directus/directus#17364] [Feature Requests]
I have a data model in Directus with Many to Many field called Games.
![Screenshot 2023-02-01 at 11 25 31](https://user-images.githubusercontent.com/96120795/215993128-e2faa43b-76d3-486f-9ca3-17cd82c64b8d.png)
I know that it is possible to order elements of the M2M field by using drag & drop but I'd like to know if it is possible to drag & drop multiple items at once?

**Comment by br41nslug** [comment]
It is not currently possible to drag n drop multiple items at once

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
